config/config.go

```diff
@@ -271,7 +281,10 @@ func Parse(cfg *Prest) {
 	cfg.SingleDB = viper.GetBool("pg.single")
 	cfg.JWTKey = viper.GetString("jwt.key")
 	cfg.JWTAlgo = viper.GetString("jwt.algo")
+	cfg.JWTWellKnown = viper.GetString("jwt.wellknown")
```

Comment:
could you rename this to `JWTWellKnownURL` ?

Reply:
Done in latest commit.
https://github.com/prest/prest/pull/871/commits/fee30e7ae1162e49e1d88f7fa384c73b016fdc95